src/price/feed.ts

```diff
@@ -60,4 +52,52 @@ export class PriceFeed {
 
     return this.collateralTokens.get('wstETH') as WstETH;
   }
+
+  private async fetchSubgraphPrice(token: CollateralToken) {
+    const query = gql`
+      query getTokenPrice($token: String!) {
+        price(id: $token) {
+          id
+          value
+          updatedAt
```

Comment:
No need to fetch other values
```suggestion
          value
```